_data/products/autonomic-resources-arc-p.json

```diff
@@ -1,22 +1,27 @@
 {
-    "slug": "autonomic-resources-arc-p",
-    "name": "Autonomic Resources ARC-P",
-    "url": "autonomicresources.com",
-    "logo_url": null,
+    "slug": "ARC-P",
+    "name": "ARC-P (IaaS)",
+    "url": "arc-p.com",
+    "logo_url": "http://autonomicresources.com/images/arc-p_logo.png"
     "top_keywords": [
         "public cloud provider gsa iaas",
         "private cloud solutions",
         "public cloud provider gsa eaas",
         "oracle services",
         "open source (products and services"
+        "ARC-P"
+        "CSRA"
+        "IaaS"
+        "Autonomic Resources"
```

Comment:
only five keywords are allowed, please choose only five, recommend against long string words unless it's part of a single topic